pkg/ddevapp/amplitude.go

```diff
@@ -0,0 +1,53 @@
+package ddevapp
```

Comment:
I usually find it annoying to have two go files with the same name, just because it's easy to get the wrong one when looking for it in the IDE.